lib/features/family/features/profiles/screens/profile_selection_screen.dart

```diff
@@ -165,38 +168,39 @@ class _ProfileSelectionScreenState extends State<ProfileSelectionScreen> {
                                 ),
                               ),
                             const SizedBox(height: 8),
-                            GivtElevatedSecondaryButton(
-                              onTap: () async {
-                                if (!context.mounted) return;
+                            if (!flow.isCoin)
```

Comment:
I understand that we don't wanna show parents on the external coin flow. Do we also intentionally want to not show the Manage My Family button? Or do we want to give people an "out" like this if they start the app from the external coin flow?

Reply:
yeah you are rught, but i am worried about potential weirdness with the flows, so i will reset the coin flow if they click it